src/components/pages/PointShare/RenderPointShare.js

```diff
@@ -1,34 +1,80 @@
+[x] component level state saves and renders correctly ---> storyOnePoints, storyTwoPoints, illustrationOnePoints, illustrationTwoPoints
+*/
+
 import React, { useEffect, useState, useCallback } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useAuth0 } from '@auth0/auth0-react';
 import { Header } from '../../common';
-import { Row, Col, InputNumber, Button, notification } from 'antd';
-import { QuestionCircleOutlined } from '@ant-design/icons';
+import Footer from '../GamePlay/Footer';
+import { Button, notification } from 'antd';
+import {
+  ZoomInOutlined,
+  CaretUpOutlined,
+  CaretDownOutlined,
+} from '@ant-design/icons';
 import { connect } from 'react-redux';
 import { submitPoints } from '../../../api/index';
 
 import { SubmissionViewerModal } from '../../common';
-import { InstructionsModal } from '../../common';
-import { modalInstructions } from '../../../utils/helpers';
+
+// import hero images - TEMPORARY HARD CODE - remove when pulling state ( avatarID ) from backend
+import childOneImg from '../../../assets/images/hero_images/hero10.png';
+import childTwoImg from '../../../assets/images/hero_images/hero8.png';
+// import sample writing submission - TEMPORARY HARD CODE - remove when pulling state from backend
+import sampleWritingSubmission from './sampleWriting.png';
+import sampleDrawingSubmission from './sampleDrawing.png';
+
+// initialize state for user object - TEMPORARY HARD CODE - remove when pulling state from backend
+const initialStateForUserTemporaryHardcode = {
+  id: 1,
+  avatarID: 1, // foreign key to avatar table
+  avatar: {
+    // avatar object is pulled into child object from avatar table (this happens on the BE)
+    id: 1,
+    fileName: 'hero10.png',
+  },
+  characterName: 'Pinky Winky',
+};
 
 const PointShare = props => {
-  const { push } = useHistory();
   const [totalPoints, setTotalPoints] = useState(100);
+
+  // these setStory and setIllustration functions set state of the 4 InputNumber components
   const [storyOnePoints, setStoryOnePoints] = useState(0);
   const [storyTwoPoints, setStoryTwoPoints] = useState(0);
   const [illustrationOnePoints, setIllustrationOnePoints] = useState(0);
   const [illustrationTwoPoints, setIllustrationTwoPoints] = useState(0);
+
   const [teamPoints, setTeamPoints] = useState(null);
   const [modalContent, setModalContent] = useState(null);
   const [showModal, setShowModal] = useState(false);
   const [modalVisible, setModalVisible] = useState(true);
 
-  const { user } = useAuth0();
+  let { user } = useAuth0();
+  let history = useHistory();
+
+  // TEMPORARY HARD CODE - remove when pulling user state from backend
+  user = initialStateForUserTemporaryHardcode;
 
   /**
-   *Virtual Player
-   *Array of virtual player IDs
-   *['VP-1','VP-2','VP-3','VP-4','VP-5','VP-6','VP-7']
+  create virtual player - TEMPORARY HARD CODE - remove when pulling state from backend
```

Comment:
Leave a comment to make explicit that this logic should be later handled by the BE.